enhancements/sandboxed-containers/sandboxed-containers-tech-preview.md

```diff
@@ -0,0 +1,620 @@
+---
+title: sandboxed-containers
+authors:
+  - "@ariel-adam"
+  - "@fidencio"
+  - "@jensfr"
+reviewers:
+  - "@cgwalters"
+  - "@mrunalp"
+approvers:
+  - "@cgwalters"
+  - "@mrunalp"
+creation-date: 2020-06-07
+last-updated: 2021-03-10
+status: implemented
+see-also:
+  - "[OpenShift sandboxed containers Operator](https://github.com/openshift/sandboxed-containers-operator)"
+  - "[CoreOS Extensions](https://github.com/openshift/enhancements/pull/317)"
+  - "[QEMU Extension](https://github.com/openshift/machine-config-operator/pull/2376)"
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+# sandboxed containers
+
+Sandboxed containers provides a container runtime using virtual machines,
+and hardware virtualization technologies, providing powerful workload
+isolation compared to the existing container solutions, with the same look
+and feel provided by vanilla containers.
+
+## Release Signoff Checklist
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in
+      [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Sumary
+
+Sandboxed containers will be integrated into OpenShift to provide the ability
+to run kernel isolated containers for any workload which requires:
+- Custom kernel tuning (sysctl, scheduler changes, cache tuning, etc).
+- Custom kernel modules (out of tree, special arguments, etc).
+- Exclusive access to hardware.
+- Root privileges.
+- Any other administrative privileges above and beyond what is secure in a
+  shared kernel environment (regular runc).
+
+Sandboxed containers relies, under the hood, on [Kata
+Containers](https://katacontainers.io/), an Open Source project able to
+achieve all the proposed points.
+
+It should be noted that Kata Containers differs from KubeVirt:
+
+- KubeVirt aims to run **VM images** on OpenShift providing the VM with the
+  look and feel of a virtual machine running as a legacy VM (CRD defining a
+  Virtual Machine in Kubernetes).
+- Kata Containers aims to run **container images** on OpenShift in an
+  isolated manner using virtualization tools (uses Runtime Class resource to
+  choose Kata Containers only for specific workloads which require kernel
+  level isolation).
+  
+## Motivation
+
+The main motivation to bring sandboxed containers to OpenShift is to provide
+an extra layer of isolation for fields that would commonly require that, such
+as Telco companies.
```

Comment:
nit: Can be modified to more security conscious users instead.

Reply:
I think the statement as makes sense as sandboxed containers provide additional isolation. Also, that would imply that security is not a default (which it is) without it.